Public/Sdk/Public/Managed/Shared/types.dsc

```diff
@@ -129,12 +129,12 @@ export interface LinkResource {
 }
 
 @@public
-export function isBinary(item: Reference) : item is Binary {
+export function isBinary(item: any) : item is Binary {

```

Comment:
>any [](start = 31, length = 3)

why do we need this?

Can we work around and keep it strongly typed?

Reply:
Needed because the runtime content to skip is a deployable item, with no explicit relation with a Reference. Align as well with the signature of isManagedPackage below, which I think looks like that for similar reasons

---
In reply to: [301268220](https://github.com/microsoft/BuildXL/pull/561#discussion_r301268220) [](ancestors = 301268220)